receiver/iisreceiver/metadata.yaml

```diff
@@ -0,0 +1,117 @@
+name: iisreceiver
+
+attributes:
+  direction:
+    value: direction
+    description: direction data is moving.
+    enum:
+      - sent
+      - received
+  request_type:
+    value: request_type
+    description: type of request sent by a client.
+    enum:
+      - delete
+      - get
+      - head
+      - options
+      - post
+      - put
+      - trace
+
+metrics:
+  iis.connection.attempt.count:
+    description: Total amount of attempts to connect to the server.
+    unit: "{attempts}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.network.io:
+    description: Total amount of bytes sent and received.
+    unit: By
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ direction ]
+  iis.request.count:
+    description: Total amount of requests of a given type.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ request_type ]
+  iis.connection.active:
+    description: Number of active connections.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.network.file.count:
+    description: Number of transmitted files.
+    unit: "{files}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+    attributes: [ direction ]
+  iis.connection.anonymous:
+    description: Number of connections established anonymously.
+    unit: "{connections}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.network.blocked:
+    description: Number of bytes blocked due to bandwidth throttling.
+    unit: By
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.request.rejected:
+    description: Total number of requests rejected.
+    unit: "{requests}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.request.queue.count:
+    description: Current number of requests in the queue.
+    unit: "{requests}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.thread.active:
+    description: Current amount of active threads.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  iis.request.queue.age.max:
+    description: Age of oldest request in the queue.
+    unit: ms
+    gauge:
+      value_type: int
+    enabled: true
```

Comment:
Would you mind rearranging these so that related namespaces are adjacent to each other?

Reply:
Updated